qiskit/scheduler/lowering.py

```diff
@@ -124,7 +124,10 @@ def get_measure_schedule(qubit_mem_slots: Dict[int, int]) -> CircuitPulseDef:
         else:
             try:
                 gate_cals = circuit.calibrations[inst.name]
-                schedule = gate_cals[(tuple(inst_qubits), tuple(float(p) for p in inst.params))]
+                schedule = gate_cals[(
+                    tuple(inst_qubits),
+                    tuple(p if getattr(p, 'parameters', None) else float(p) for p in inst.params),
```

Comment:
Can you please add a test where this would have failed? Please also add the following test:
```
x = Parameter('x')

circ = QuantumCircuit(2, 2)
circ.append(Gate('custom', 1, [x]), [0])
circ.assign_parameters({x: 1})
circ.add_calibration('custom', [0], Schedule(), [1])

sched = schedule(circ, backend)
```
I believe this was first added because `inst.params[0]` will still be `x` and won't match `1`, even though the value of `x` is `1`.